airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/setup_and_teardown_runner.py

```diff
@@ -17,16 +17,15 @@
 async def _build_container(dagger_client: dagger.Client, dockerfile_path: Path) -> dagger.Container:
     workspace = (
         dagger_client.container()
-        .with_directory("/tmp", dagger_client.host().directory(os.path.dirname(dockerfile_path)))
-        .with_workdir("/tmp")
-        .with_file("/tmp/Dockerfile.cat_setup_teardown", dagger_client.host().file(str(dockerfile_path.expanduser())))
-        .directory("/tmp")
+        .with_mounted_directory("/tmp/setup_teardown_context", dagger_client.host().directory(os.path.dirname(dockerfile_path)))
```

Comment:
You might want to exclude some content from the connector directory as it might be costly to mount everything. Feel free to copy some logic we have in airbyte-ci for that [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/models/contexts/pipeline_context.py#L49)